lib/app_profiler/yarn/command.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+module AppProfiler
+  module Yarn
+    module Command
+      class YarnError < StandardError; end
+
+      mattr_accessor :yarn_setup, default: false
+
+      def yarn(command)
+        setup_yarn unless yarn_setup
+        exec("yarn #{command}") do
+          raise YarnError, "Failed to run #{command}."
+        end
+      end
```

Comment:
This seems very dangerous. Would highly recommend we not just accept any command pass it to yarn. I don't know it well enough but if there's any flag that yarn takes that could result in arbitrary read/write, it would be bad. Would recommend instead we allow-list the commands or actual write the methods with the allowed yarn commands we'd accept

Reply:
This was already in the codebase, I just moved it. Will address in a separate commit.